### PR TITLE
Run consensus tests with real Byron blocks

### PR DESCRIPTION
**Pull request by mrBliss**
- [X] Generate transactions for Byron blocks, **postponed:** #888 
- [X] Produce a genesis EBB, **this was happening already**

### CODE REVIEW

**Comment by mrBliss** [comment]
@ruhatch is going to help us with the transaction generation for Byron blocks. Feel free to push to this branch.

File: ouroboros-consensus/test-consensus/Test/Dynamic/RealPBFT.hs [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+    dlgMap :: Map Common.KeyHash Delegation.Certificate
+    dlgMap = Genesis.unGenesisDelegation
+           $ Genesis.gdHeavyDelegation
+           $ Genesis.configGenesisData genesisConfig
+
+
+-- Instead of using 'Dummy.dummyConfig', which hard codes the number of rich
+-- men (= CoreNodes for us) to 4, we generate a dummy config with the given
+-- number of rich men.
```
Comment (edsko) [inline ouroboros-consensus/test-consensus/Test/Dynamic/RealPBFT.hs]: Did this go wrong in the case where the rich men were too few?

Comment (mrBliss) [reply ouroboros-consensus/test-consensus/Test/Dynamic/RealPBFT.hs]: Indeed, with `Dummy.dummyConfig`:
```
*** Failed! Exception: 'Prelude.!!: index too large' (after 9 tests and 6 shrinks):
NumCoreNodes 5
...
```

**Comment by mrBliss** [comment]
*pets bors*. I knew this would fail, even though CI wasn't warning us.

**Comment by mrBliss** [comment]
Come on bors, you're holding us back :disappointed:

**Comment by edsko** [comment]
Shouldn't have petted it.

**Comment by mrBliss** [comment]
Ah, there you are, bors!

**Pull request merged**